src/sql/workbench/parts/objectExplorer/electron-browser/objectExplorerScripting.contribution.ts

```diff
@@ -71,41 +76,61 @@ MenuRegistry.appendMenuItem(MenuId.ObjectExplorerItemContext, {
 		ContextKeyExpr.or(
 			ContextKeyExpr.and(
 				ConnectionContextKey.Provider.isEqualTo('MSSQL'),
-				TreeNodeContextKey.NodeType.isEqualTo('StoredProcedure')),
+				TreeNodeContextKey.NodeType.isEqualTo(NodeType.StoredProcedure)),
 			ContextKeyExpr.and(
 				ConnectionContextKey.Provider.isEqualTo('MSSQL'),
-				TreeNodeContextKey.NodeType.isEqualTo('View')),
+				TreeNodeContextKey.NodeType.isEqualTo(NodeType.View)),
 			ContextKeyExpr.and(
 				ConnectionContextKey.Provider.isEqualTo('MSSQL'),
-				TreeNodeContextKey.NodeType.isEqualTo('AggregateFunction')),
+				TreeNodeContextKey.NodeType.isEqualTo(NodeType.AggregateFunction)),
 			ContextKeyExpr.and(
 				ConnectionContextKey.Provider.isEqualTo('MSSQL'),
-				TreeNodeContextKey.NodeType.isEqualTo('PartitionFunction')),
+				TreeNodeContextKey.NodeType.isEqualTo(NodeType.PartitionFunction)),
 			ContextKeyExpr.and(
 				ConnectionContextKey.Provider.isEqualTo('MSSQL'),
-				TreeNodeContextKey.NodeType.isEqualTo('ScalarValuedFunction')),
+				TreeNodeContextKey.NodeType.isEqualTo(NodeType.ScalarValuedFunction)),
 			ContextKeyExpr.and(
 				ConnectionContextKey.Provider.isEqualTo('MSSQL'),
-				TreeNodeContextKey.NodeType.isEqualTo('TableValuedFunction')),
+				TreeNodeContextKey.NodeType.isEqualTo(NodeType.TableValuedFunction)),
 		)
 });
 
 MenuRegistry.appendMenuItem(MenuId.ObjectExplorerItemContext, {
 	group: 'scripting',
-	order: 3,
+	order: 4,
 	command: {
 		id: SCRIPT_AS_DELETE_COMMAND_ID,
 		title: localize('scriptDelete', "Script as Drop")
 	},
 	when: ContextKeyExpr.or(
-		TreeNodeContextKey.NodeType.isEqualTo('Table'),
-		TreeNodeContextKey.NodeType.isEqualTo('View'),
-		TreeNodeContextKey.NodeType.isEqualTo('Schema'),
-		TreeNodeContextKey.NodeType.isEqualTo('User'),
-		TreeNodeContextKey.NodeType.isEqualTo('UserDefinedTableType'),
-		TreeNodeContextKey.NodeType.isEqualTo('StoredProcedure'),
-		TreeNodeContextKey.NodeType.isEqualTo('AggregateFunction'),
-		TreeNodeContextKey.NodeType.isEqualTo('PartitionFunction'),
-		TreeNodeContextKey.NodeType.isEqualTo('ScalarValuedFunction'),
-		TreeNodeContextKey.NodeType.isEqualTo('TableValuedFunction'))
+		TreeNodeContextKey.NodeType.isEqualTo(NodeType.Table),
+		TreeNodeContextKey.NodeType.isEqualTo(NodeType.View),
+		TreeNodeContextKey.NodeType.isEqualTo(NodeType.Schema),
+		TreeNodeContextKey.NodeType.isEqualTo(NodeType.User),
+		TreeNodeContextKey.NodeType.isEqualTo(NodeType.UserDefinedTableType),
+		TreeNodeContextKey.NodeType.isEqualTo(NodeType.StoredProcedure),
+		TreeNodeContextKey.NodeType.isEqualTo(NodeType.AggregateFunction),
+		TreeNodeContextKey.NodeType.isEqualTo(NodeType.PartitionFunction),
+		TreeNodeContextKey.NodeType.isEqualTo(NodeType.ScalarValuedFunction),
+		TreeNodeContextKey.NodeType.isEqualTo(NodeType.TableValuedFunction))
+});
+
+MenuRegistry.appendMenuItem(MenuId.ObjectExplorerItemContext, {
+	group: 'scripting',
```

Comment:
I think scripting should be in it's own group

Reply:
If it's in its own group then we get the separator which we didn't have earlier

Reply:
In your screenshot we have the separator anyways between refresh and properties. I don't think the separator is an issue @kburtram

Reply:
the separator isn't a problem (either being present or not).